src/main/java/AddCommand.java

```diff
@@ -0,0 +1,35 @@
+public class AddCommand extends Command {
+
+    public AddCommand(String action, String info) {
+        super(action, info);
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        String task, time;
+        int size;
+        Task t;
```

Comment:
I think you can select a better variable name than t, as the scope here is for the whole method.

Reply:
Thanks for the input.